### PR TITLE
Add IANA port for P4RT to default external port list

### PR DESCRIPTION
**Pull request by pudelkoM**
IANA allocated TCP port 9559 for P4RT. This PR adds it to the default external urls that Stratum exposes.

TODO:

- [x] Document port usage

Upstream issue: https://github.com/p4lang/p4runtime/issues/306

### CODE REVIEW

**Review by Yi-Tseng — DISMISSED** [review]
lgtm, and it is now official

https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?search=9559

File: stratum/lib/constants.h [hunk 1]
```diff
@@ -34,8 +33,10 @@ constexpr char kDefaultAuthPolicyFilePath[] =
 constexpr char kLocalStratumUrl[] = "localhost:28000";
 
 // This URL is used by external gNMI, gNOI and P4Runtime clients.
-// This TCP port 9339 is an IANA-reserve port for gNMI and gNOI.
-constexpr char kExternalStratumUrls[] = "0.0.0.0:28000,0.0.0.0:9339";
+// This TCP port 9339 is an IANA-reserved port for gNMI and gNOI.
```
Comment (stolsma) [inline stratum/lib/constants.h:36]: Super minor nit...  Maybe also a remark to port 28000 can be added also explaining that it is deprecated and will be removed in the near future?  Also maybe the 'This' from 'This TCP port 9339' can be removed to be consistent with the line below?

Comment (pudelkoM) [reply stratum/lib/constants.h:36]: We'll do the deprec notice in the release notes too.

**Comment by stolsma** [comment]
Thanks Max!

File: stratum/hal/bin/np4intel/docker/README.md [hunk 2]
```diff
@@ -198,7 +198,7 @@ to debug problems using gdb and the source code.
 ```
 
 5. At this point the ptf framework can be used to load tables or a real
-control plane can be started to talk to Stratum on port 28000.  If it's
+control plane can be started to talk to Stratum on port 9559. If it's
 on the local machine it will likely be localhost:28000 (depends on your
```
Comment (bocon13) [inline stratum/hal/bin/np4intel/docker/README.md:202]: 28000?

**Review by bocon13 — APPROVED** [review]

**Pull request merged**